test/functionalTest/cases/0999_update_single_attr_value_by_eid/update_single_attr_value_by_eid_fake_field_cases.test

```diff
@@ -21,7 +21,7 @@
 # VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
 
 --NAME--
-Error handling For PUT Individual Entity Attribute
+PUT Entity Attribute value fake value and type cases
```

Comment:
Not sure if "fake fields (fake value, fake type)" is the right term... What I mean is using values that use the keywords "value" and "type" that make sense in other operations (e.g. POST /v2/entities without keyValues option). If somebody propose a better term, let's use it.